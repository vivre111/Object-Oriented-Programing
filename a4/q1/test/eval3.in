a 1 + ABS NEG b - ABS c * done
print
eval
set a 3
print
set b 5
set c 9
eval
print
