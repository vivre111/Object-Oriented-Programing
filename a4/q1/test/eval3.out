(|(-|(a + 1)| - b)| * c)
(|(-|(a + 1)| - b)| * c)
a has no value.
(|(-|(3 + 1)| - b)| * c)
81
(|(-|(3 + 1)| - 5)| * 9)
